### ISSUE TITLE
Dataset Natural Disasters: Column Duration, which unit for time is more adecuate?

### ISSUE THREAD

**Issue by variableVG** [FreshJonas/Nat_Disasters_vs_GDP#1]
Most natural disasters lasted less than a year (15 522 natural disaster). Perhaps it would be more interesting to know how many months/days they lasted.

Perhaps it could be interesting to know also, in which year they started and how that affected to the GDP that year.

**Comment by FreshJonas** [comment]
I Agree. Days would probably make the most sense since you can always deal with integers that way. Feel free to implement that change
